packages/theme/components/layout/article/Article.module.css

```diff
@@ -1,42 +1,38 @@
 .Article {
   display: grid;
+  grid-template-areas: 'main';
+  grid-template-columns: 1fr;
 }
 
 .Article--withToc {
   gap: var(--base-size-48);
+  grid-template-areas: 'main aside';
+  grid-template-columns: 1fr 200px;
 }
 
 .main {
-  order: 1;
+  grid-area: main;
+  max-width: 100%;
+  overflow-x: hidden;
 }
 
 .aside {
-  order: 0;
+  grid-area: aside;
+  position: relative;
 }
 
-@media screen and (max-width: 48rem) {
+@media screen and (max-width: 1023px) {
```

Comment:
In hindsight it's pretty odd that we don't have an equivalent size for this in Primer Brand 🤔 

Guessing you can't use `64rem` here Dani? Would convert to 1024px. Or `63.938rem` to at least be consistent in using rem (converts to 1023px)

Reply:
I'm happy to go with either option. We previously used `1023px` a few lines below, and I found it clearer than using a float rem like `63.938rem`. Let me know what you prefer, and I can make the update.